versions.yaml

```diff
@@ -12,7 +12,7 @@ gh: 2.62.0
 gojq: v0.12.17
 
 # protobuf formatters/plugins/tools/etc
-buf: 1.8.0 # formatter
```

Comment:
Deleted the comment because it will be more than a formatter starting with #1009